components/workspace/FilterSelector.tsx

```diff
@@ -17,6 +20,22 @@ const toggleItemCheck = (item: string, select: boolean = true) => {
   }
 }
 
+const matchMinimumAccountAge = (
+  minAccountAgeUnit: string,
```

Comment:
Would it be possible/reasonable to type this as `'days' | 'hours'`?

Reply:
we could but it's a bit dependent on typing the whole state object which im not doing right now and im not seeing much benefit of doing that right now.